src/Surfnet/StepupSelfService/SelfServiceBundle/Service/RemoteVetting/Dto/AttributeListDto.php

```diff
@@ -18,17 +18,20 @@
 
 namespace Surfnet\StepupSelfService\SelfServiceBundle\Service\RemoteVetting\Dto;
 
+use phpDocumentor\Reflection\Types\Self_;
```

Comment:
Was this intentionally?

Reply:
Good catch. No that was added by mistake. Also removed the stale `AttributeMatchCollection`